cmake/FindLibLUV.cmake

```diff
@@ -4,9 +4,15 @@
 #  LIBLUV_INCLUDE_DIRS - The libluv include directories
 #  LIBLUV_LIBRARIES - The libraries needed to use libluv
 
+find_package(PkgConfig)
+if (PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_LIBTERMKEY QUIET luv)
```

Comment:
what relationship LIBTERMKEY and luv?

Reply:
Typo..!
Thanks, fixed.
@mhinz 
Please try again.